wavefront-istio-adapter/Chart.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+name: wavefront-istio-adapter
+version: 0.1.3
```

Comment:
Also add an `appVersion` which refers to the version of the Adapter. This one refers to the version of the chart itself which can differ based on changes we make to the Helm charts.